### DISCUSSION TITLE
Website link without protocol should behave like absolute

### DISCUSSION THREAD

**Discussion by ftoh** [community/community#3178] [Discussions]
In [cmder](https://github.com/cmderdev/cmder) repository the link follow to `http://cmder.net/`
but in [discussions](https://github.com/cmderdev/cmder/discussions) it follow to `https://github.com/cmderdev/cmder/cmder.net`

**Comment by PranamBhat** [comment]
Hi @FTOH 

It is just a tying error.

Copy your link (which you want to redirect) and paste it in the discussion.

Now select the link and click on "**Add a link (ctrl+k)**" and inside the squre brackets [] , paste your link.

For example:

[**Link**](https://cmder.net/)

**Reply by ftoh** [reply]
Hi, I meant the link that is specified in the repository settings.
![image](https://user-images.githubusercontent.com/5018813/108165558-5a70c280-7125-11eb-8fe3-d638fa2fcfb7.png)
![image](https://user-images.githubusercontent.com/5018813/108165609-6c526580-7125-11eb-8b25-18361e9e7e73.png)

**Comment by PranamBhat** [comment]
Got it.

It's because, you have given wrong link address while updating:

![image](https://user-images.githubusercontent.com/32120033/108174525-8b50f780-7125-11eb-8d52-865765fbdd55.png)


It should be 

![image](https://user-images.githubusercontent.com/32120033/108174413-69f00b80-7125-11eb-94fa-c6e93c324877.png)
